src/TRestRawSignalRemoveChannelsProcess.cxx

```diff
@@ -143,24 +145,55 @@ void TRestRawSignalRemoveChannelsProcess::LoadConfig(const string& configFilenam
 ///
 TRestEvent* TRestRawSignalRemoveChannelsProcess::ProcessEvent(TRestEvent* inputEvent) {
     fInputSignalEvent = (TRestRawSignalEvent*)inputEvent;
+    fInputSignalEvent->InitializeReferences(GetRunInfo());
 
-    for (int n = 0; n < fInputSignalEvent->GetNumberOfSignals(); n++) {
-        TRestRawSignal* sgnl = fInputSignalEvent->GetSignal(n);
+    if (fReadoutMetadata == nullptr) {
+        fReadoutMetadata = fInputSignalEvent->GetReadoutMetadata();
+    }
 
-        Bool_t removeChannel = false;
-        for (unsigned int x = 0; x < fChannelIds.size() && !removeChannel; x++)
-            if (sgnl->GetID() == fChannelIds[x]) removeChannel = true;
+    if (fReadoutMetadata == nullptr) {
+        std::cerr << "TRestRawBaseLineCorrectionProcess::ProcessEvent: readout metadata is null" << std::endl;
+        exit(1);
+    }
 
-        if (!removeChannel) fOutputSignalEvent->AddSignal(*sgnl);
+    for (int n = 0; n < fInputSignalEvent->GetNumberOfSignals(); n++) {
+        TRestRawSignal* sgnl = fInputSignalEvent->GetSignal(n);
```

Comment:
In general I think it's better to avoid variable names such as `sgnl` (which are not real words) when you can use the full-length word `signal`.

I know you didn't come up with the idea and it was this way before, but this is a good opportunity to update all these small things if you agree.